Libplanet/Store/LiteDBStore.cs

```diff
@@ -376,6 +376,12 @@ public override IEnumerable<HashDigest<SHA256>> IterateBlockHashes()
         /// <inheritdoc/>
         public override void PutBlock<T>(Block<T> block)
```

Comment:
Since the behavior of the method has changed, it should be written to the changelog and method documentation.

Could you add this to the changelog behavioral change section?
```
 -  `IStore.PutBlock<T>()` became to do nothing when it takes
    the `Block<T>` more than once.  [[#647]]
```

And could you add this to the `IStore.PutBlock<T>` documentation summary section?
```
If the same block already exists in the store it does nothing.
```

Reply:
Ok